css-shared-element-transitions/index.bs

```diff
@@ -593,254 +655,307 @@ document's DOM structure.
-    
-            Issue: Which of ''xywh()''/''rect()''/''inset()'' should we use?
-
-		1. If |CapturedData| has an [=incoming element=],
-			then create be a new ''::page-transition-incoming-image''
-			pseudo-element
-			with the tag |tag|,
-			inserted as a child of |image wrapper|
-			(after |outgoing|, if it exists), if it doesn't exist.
-			Let |incoming| be the ''::page-transition-incoming-image'' pseudo-element with
-			the tag |tag|.
-			This pseudo-element is a [=replaced element=],
-			displaying the [=capture the image=]
-			of |CapturedElement|&apos;s [=incoming element=].
-
-			At the [=user-agent origin=],
-			set |incoming|&apos;s 'object-view-box' property
-			to a value that when applied to |incoming|,
-			will cause the view box to coincide with [=incoming element=]'s [=border box=]
-			in the image.
+    To <dfn>create transition pseudo-elements</dfn> for a {{SameDocumentTransition}} |transition|:
+
+    1. Let |transitionRoot| be the result of creating a new ''::page-transition'' pseudo-element.
+
+    1. [=map/For each=] |transitionTag| → |capturedElement| of |transition|'s [=SameDocumentTransition/tagged elements=]:
+
+        1. Let |container| be the result of creating a new ''::page-transition-container'' pseudo-element with the tag |transitionTag|.
+
+            Issue: "tag" should be defined/linked.
+
+        1. Append |container| to |transitionRoot|.
+
+            Issue: This should be better defined.
+                I'm not sure if pseudo-elements have defined ways to modify their DOM.
+
+        1. Let |width|, |height|, |transform|, |writingMode|, and |direction| be null.
+
+        1. If |capturedElement|'s [=incoming element=] is null, then:
+
+            1. Set |width| to |capturedElement|'s [=outgoing styles=] 'width' property.
+
+            1. Set |height| to |capturedElement|'s [=outgoing styles=] 'height' property.
+
+            1. Set |transform| to |capturedElement|'s [=outgoing styles=] 'transform' property.
+
+            1. Set |writingMode| to |capturedElement|'s [=outgoing styles=] 'writing-mode' property.
+
+            1. Set |direction| to |capturedElement|'s [=outgoing styles=] 'direction' property.
+
+        1. Otherwise:
+
+            1. Set |width| to the current width of |capturedElement|'s [=incoming element=]'s [=border box=].
+
+            1. Set |height| to the current height of |capturedElement|'s [=incoming element=]'s [=border box=].
+
+            1. Set |transform| to a transform that maps the |capturedElement|'s [=incoming element=]'s [=border box=] from document origin to its quad in viewport.
+
+            1. Set |writingMode| to the [=computed value=] of 'writing-mode' on |capturedElement|'s [=incoming element=].
+
+            1. Set |direction| to the [=computed value=] of 'direction' on |capturedElement|'s [=incoming element=].
+
+        1. Let |width| be the current width of |capturedElement|'s [=incoming element=]'s [=border box=],
```

Comment:
Did you mean to delete these 3 points. It's already covered by the if else block above.

Reply:
oof, yes. Good catch.